core/src/main/java/nu/yona/server/device/service/DeviceAnonymizedDto.java

```diff
@@ -19,25 +21,36 @@
 	private final UUID id;
 	private final int deviceIndex;
 	private final OperatingSystem operatingSystem;
+	private final LocalDate lastMonitoredActivityDate;
+	private final String appVersion;
+	private final int appVersionCode;
 	private final String vpnLoginId;
+	private final boolean isVpnConnected;
 	private final String firebaseInstanceId;
 	private final Locale locale;
 
-	public DeviceAnonymizedDto(UUID id, int deviceIndex, OperatingSystem operatingSystem, String vpnLoginId,
-			Optional<String> firebaseInstanceId, Locale locale)
+	public DeviceAnonymizedDto(UUID id, int deviceIndex, OperatingSystem operatingSystem,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 10 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00107)